cli-tests/test/test.bats

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env bats
+
+setup_file() {
```

Comment:
Is there a more descriptive name for this method like "setup_binary_file", "retrieve_script" or "unzip_script"?

Reply:
It's part of the framework https://bats-core.readthedocs.io/en/stable/tutorial.html#avoiding-costly-repeated-setups It has to be that.